src/components/Search/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { HiSearch } from 'react-icons/hi';
+import { IconContext } from 'react-icons';
+
+type Props = {
+  keyword: string;
+  setKeyword: (value: string) => void;
+};
+
+const Search = ({ keyword, setKeyword }: Props) => {
+  return (
+    <div className="flex flex-1 relative block cursor-text">
+      <input
+        type="text"
+        name="search"
+        id="search"
+        title="Search for a recording"
```

Comment:
For other projects where a Search component might be reused, it'll be ideal for the title/placeholder to be a prop. Otherwise, I think it's fine to leave it as is for this project.